packages/oidc-client/src/initWorker.ts

```diff
@@ -106,7 +106,7 @@ export const initWorkerAsync = async (configuration, configurationName) => {
   }
 
   let registration = null;
-  if (configuration.register) {
+  if (configuration.register || configuration.service_worker_register) {
```

Comment:
thank you so much @thiscode 
It should be only
if (configuration.service_worker_register) {

I have checkout the code and configuration.register does not exist any more. I should add more Typing to this file.

Reply:
I squash merge your PullRequest, Thank you @thiscode ! it is already better.